mdm/mdm/command.go

```diff
@@ -17,7 +17,7 @@ type CommandPayload struct {
 
 func NewCommandPayload(request *CommandRequest) (*CommandPayload, error) {
 	payload := &CommandPayload{
-		CommandUUID: uuid.New().String(),
+		CommandUUID: request.CommandUUID,
```

Comment:
See below for notes on where `uuid.New()` should go and also whether this ought to be a pointer or not (I'd lean to no).